frontend/src/lib/components/LemonButton/LemonButton.scss

```diff
@@ -24,6 +24,15 @@
         cursor: not-allowed;
     }
 
+    &.LemonButton--unclickable {
+        cursor: not-allowed;
+        padding-bottom: 0.5rem;
+        padding-top: 0.5rem;
+        max-width: 20rem;
+        opacity: 100%;
+        background: var(--side);
+    }
+
```

Comment:
This feels a bit hacky to me. What we want to add is a way to have an option in the select menu that is not an option but rather a customisable section (just like how you can customise the section title). This is already somewhat supported for titles, we just need to modify it to allow either a customisable footer or options in general
https://storybook.posthog.net/?path=/docs/lemon-ui-lemon-select--sectioned-options#sectioned-options